docs/coding-guidelines/breaking-change-rules.md

```diff
@@ -169,6 +169,8 @@ Breaking Change Rules
 
 * Changing a `struct` type to a `ref struct` type and vice versa
 
+* Adding a new `ref` field to a type
```

Comment:
Would something along these lines be more precise?

```suggestion
* Adding a new `ref` field to a type that didn't have any `ref` field before (recursively)
```

Reply:
Yes, that is much clearer and actually the breaking change issue. Thanks.